Samples/iOS-SwiftUI/iOS-SwiftUI/SwiftUIApp.swift

```diff
@@ -5,13 +5,32 @@ import SwiftUI
 
 @main
 struct SwiftUIApp: App {
-    init() {
-        SentrySDKWrapper.shared.startSentry()
-    }
-    
+    @UIApplicationDelegateAdaptor private var appDelegate: MyAppDelegate
+
     var body: some Scene {
         WindowGroup {
             ContentView()
         }
     }
 }
+
+class MyAppDelegate: NSObject, UIApplicationDelegate, ObservableObject {
+    func application(_ application: UIApplication, configurationForConnecting connectingSceneSession: UISceneSession, options: UIScene.ConnectionOptions) -> UISceneConfiguration {
+        let configuration = UISceneConfiguration(
+            name: nil,
+            sessionRole: connectingSceneSession.role)
+        if connectingSceneSession.role == .windowApplication {
+            configuration.delegateClass = MySceneDelegate.self
+        }
+        return configuration
+    }
+}
+
+class MySceneDelegate: NSObject, UIWindowSceneDelegate, ObservableObject {
+    var initializedSentry = false
+    func sceneDidBecomeActive(_ scene: UIScene) {
+        guard !initializedSentry else { return }
+        SentrySDKWrapper.shared.startSentry()
```

Comment:
`h`: We should instead do something like `SentrySDKWrapper.shared.installDebugMenu()` and keep the initialization of the SDK in the init of the App